objects/key.c

```diff
@@ -38,7 +38,7 @@
 #include "common/xutil.h"
 #include "xkb.h"
 
-/* XStringToKeysym() and XKeysymToString */
+/* XStringToKeysym() */
 #include <X11/Xlib.h>
```

Comment:
One of the main reasons xcb doesn't have this yet, BTW, was the fact that xcb had no xrm equivalent. It should now be portable to XCB according to a discussion from 2008. :-)